src/encord_active/lib/model_predictions/reader.py

```diff
@@ -23,13 +25,38 @@ class OntologyObjectJSON(TypedDict):
     color: str
 
 
+class OntologyClassificationJSON(TypedDict):
```

Comment:
If you like my other suggestion, this definition needs to go to the writer to avoid cyclic imports.